README.md

```diff
@@ -66,7 +66,21 @@ file.csslint.results = []; // CSSLint errors
 file.csslint.opt = {}; // The options you passed to CSSLint
 ```
 
-## Custom Reporters
+## Using reporters
+
+Using built-in reporters are quite easy. First pipe your css-files in `gulp-csslint`, then pipe them through `csslint.reporter`, specifying which reporter you'd like to use.
+
+For a list of all reporters supported by `csslint`, see the [csslint wiki](https://github.com/CSSLint/csslint/wiki/Command-line-interface#--format).
+
+```js
+gulp.task('lint', function() {
+  gulp.files('lib/*.css')
+    .pipe(csslint())
+    .pipe(csslint.reporter('junit-xml'));
```

Comment:
@joshuacc we've removed the ability to write the output to a file for now so we can merge the PR. I'll talk to some folks and see if we can come up with a smart way of doing this that's consistent between gulp plugins.